tensorflow/python/keras/optimizer_v2/learning_rate_schedule.py

```diff
@@ -537,6 +537,7 @@ def __init__(
       self,
       initial_learning_rate,
       decay_steps,
+      warmup_steps=0,
```

Comment:
do you have a good example of where this is used? API changes are non-trivial and needs to be closely examined.